config/prod/newinstance_1.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
+stack_name: jb-new-instance-1
```

Comment:
could you please change filename to match stack_name?  thanks.

Reply:
Done!